invenio_madmp/config.py

```diff
@@ -6,3 +6,49 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """Invenio module for maDMP integration."""
+
+from typing import Dict, List
+
+from .convert.records.base import BaseRecordConverter
+from .convert.records.rdm_records import RDMRecordConverter
+from .licenses import KNOWN_LICENSES
+
+# for finding matching distributions
+MADMP_HOST_URL = "https://data.tuwien.ac.at"
```

Comment:
In order to run the config variables [readthedocs](https://invenio-madmp.readthedocs.io/en/latest/configuration.html), the comment should be something like this suggestion.
```suggestion
# for finding matching distributions
MADMP_HOST_URL = "https://data.tuwien.ac.at"
"""Host URL for MADMPs."""
```

Reply:
created issue #9 for this